### ISSUE TITLE
Bookmarks and hyperlink within the document (ODT) not getting converted into appropriate PDF behaviour

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#114]
```
What steps will reproduce the problem?

1. Use the attached ODT sample file. It has 2 pages. The first page has a link 
to the next page having a book mark. Clicking on the link takes you to the next 
page in openoffice.

2. Convert the ODT to PDF running the appached ARunner.java code.


Expected: The output pdf behaves exactly as the ODT. On clicking "page", the 
next page is opened in Adobe Acrobat reader.
Actual: The link is displayed in blue for the text "page", but clicking it does 
not navigate to that page.

Version:0.9.7
OS: Windows 7 (64 bit)

Thanks,
Shreyas





```

Original issue reported on code.google.com by `shreyas....@gmail.com` on 2 May 2012 at 4:07

Attachments:
- [LinkTest_Tmpl.odt](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-114/comment-0/LinkTest_Tmpl.odt)
- [ARunner.java](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-114/comment-0/ARunner.java)
- [LinkTest_Out.pdf](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-114/comment-0/LinkTest_Out.pdf)

**Comment by GoogleCodeExporter** [comment]
```
Your problem is that ODT converter to PDF doeesn't manage well hyperlink. I 
think I had not finished my work about hyperlink converstion.

@dLeszek could you please see this problem?

Thank a lot.

Regards Angelo
```

Original comment by `angelo.z...@gmail.com` on 2 May 2012 at 5:52
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Ok, I can take a look at it
```

Original comment by `abe...@gmail.com` on 4 May 2012 at 7:22

**Comment by GoogleCodeExporter** [comment]
```
Many thanks Leszek !
```

Original comment by `angelo.z...@gmail.com` on 4 May 2012 at 12:34

**Comment by GoogleCodeExporter** [comment]
```
fixed in version 0.9.8
```

Original comment by `abe...@gmail.com` on 18 May 2012 at 11:20
- Changed state: **Fixed**
